tkg/client/init.go

```diff
@@ -231,6 +231,17 @@ func (c *TkgClient) InitRegion(options *InitRegionOptions) error { //nolint:funl
 		if err = c.InstallOrUpgradeManagementComponents(bootStrapClusterClient, bootstrapPkgClient, "", false); err != nil {
 			return errors.Wrap(err, "unable to install management components to bootstrap cluster")
 		}
+
+		akoRequired, err := c.isAKORequiredInBootstrapCluster()
+		if err != nil {
+			return errors.Wrap(err, "unable to check whether avi ha is enabled")
+		}
+		if akoRequired {
+			log.Info("Installing AKO...")
+			if err = c.InstallAKO(bootStrapClusterClient); err != nil {
+				return errors.Wrap(err, "unable to upgrade ako")
```

Comment:
The error message here should be `unable to install ako on bootstrap cluster`, right?
And I also think since we are installing ako on bootstrap cluster, maybe it is better to change the log info `"Installing AKO..."` to `Installing AKO on bootstrap cluster...` to be the same log patterns of other bootstrap cluster components. This can also reduce some confusions.

Reply:
Let me fix it